spec/features/navigation/navigation_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'Site Navigation' do
+  before :each do
+
+    visit '/'
+
+		click_link 'Register'
+
+		expect(current_path).to eq("/register")
+
+		name = "Happy Gilmore"
+		street_address = "45433 Fake st."
+		city = "Denver"
+		state = "Colorado"
+		zip = 80234
+		email = "strangerthings@gmail.com"
+		password = "BestFakerEver"
+		password_confirmation = "BestFakerEver"
+
+    fill_in :name, with: name
+    fill_in :street_address, with: street_address
+    fill_in :city, with: city
+    fill_in :state, with: state
+    fill_in :zip, with: zip
+    fill_in :email, with: email
+    fill_in :password, with: password
+    fill_in :password_confirmation, with: password_confirmation
+
+    click_on 'Create User'
+  end
+
+  it 'shows this information from the nav bar' do
+      within('#top-nav') do
+        expect(page).to have_link("Home")
```

Comment:
Here I am just checking that everything from the user story below can be found in the navbar (application.html in views/layouts). It already had the merchants and the items links up there so I just add the extra ones as well as the id in the navbar class so we could test within the nav bar. Since it is in the navbar it should be on every page.